src/sentry/static/sentry/app/views/events/searchBar.jsx

```diff
@@ -72,14 +72,29 @@ class SearchBar extends React.PureComponent {
     ({key}, query) => `${key}-${query}`
   );
 
-  getAllTags = (orgTags = []) => orgTags.sort().reduce(tagToObjectReducer, {});
-
   /**
    * Prepare query string (e.g. strip special characters like negation operator)
    */
   prepareQuery = query => query.replace(SEARCH_SPECIAL_CHARS_REGEXP, '');
 
+  getTagList() {
+    const {organization, tags} = this.props;
+    const fields = organization.features.includes('transaction-events')
+      ? FIELD_TAGS
+      : omit(FIELD_TAGS, TRACING_FIELDS);
+    const combined = assign({}, tags, fields);
+    combined.has = {
+      key: 'has',
+      name: 'Has property',
+      values: Object.keys(combined),
+      predefined: true,
+    };
```

Comment:
This could be part of `assign()` 🚵‍♂️

```suggestion
    const combined = assign({}, tags, fields, {
    has: {
      key: 'has',
      name: 'Has property',
      values: Object.keys(combined),
      predefined: true,
    }});
```